virttest/sriov.py

```diff
@@ -0,0 +1,110 @@
+"""
+SRIOV related utility functions
+"""
+
+import logging
+
+from avocado.core import exceptions
+
+from virttest import utils_misc
+from virttest import utils_net
+from virttest import remote
+from virttest.staging import service
+
+
+def find_pf(driver, session=None):
+    """
+    Check if available PF exists to test
+
+    :param session: The session object to the host
+    :raise: test.cancel when no pf found
```

Comment:
generally, we don't have test.cancel available, but just exception in utils module